libraries/AP_InertialSensor/AP_InertialSensor.cpp

```diff
@@ -1495,7 +1481,7 @@ void AP_InertialSensor::update(void)
     // wait_for_sample(), and a wait is implied
     wait_for_sample();
 
-    if (!_hil_mode) {
+    if (true) {
```

Comment:
not useful

Reply:
I could instead remove the brackets and leave it indented...

Reply:
Removed it but did not reindent the very substantial block